source/smart-contracts/tutorials/piggy-bank/writing.rst

```diff
@@ -84,25 +84,27 @@ Piggy bank contract
 ===================
 
 The contract we are going to build in this tutorial is going to act as a classic
-piggy bank. Everyone should be able to insert GTU into it, but only the owner
+piggy bank. Everyone should be able to put money into it, but only the owner
```

Comment:
I think we need to use GTU and not money.

We are specifically using the dedicated amount to contribute and using the chain to keep track of it, and the chain keeps track of GTU only.

Reply:
```suggestion
piggy bank. Everyone should be able to put money in the form of GTU into it, but only the owner
```
Sounds good to me--I'd like to use money in the first use of this because a piggy bank usually contains money so I want to use that word just once, and below I'll say GTU.

Reply:
Ok.